e2e/node_config.go

```diff
@@ -653,6 +653,10 @@ func baseTemplateLinux(t *testing.T, location string, k8sVersion string, arch st
 					GalleryName:   "AKSUbuntuEdgeZone",
 					ResourceGroup: "AKS-Ubuntu-EdgeZone",
 				},
+				"AKSFlatcar": {
```

Comment:
SIGConfig is entirely unused in agentbaker e2e, though thanks for being consistent

Reply:
it looks unused, but if it can't be populated correctly e2e fails with:
```
    vmss.go:52: 
        	Error Trace:	/mnt/vss/_work/1/s/e2e/vmss.go:52
        	            				/mnt/vss/_work/1/s/e2e/test_helpers.go:153
        	            				/mnt/vss/_work/1/s/e2e/test_helpers.go:92
        	            				/mnt/vss/_work/1/s/e2e/scenario_win_test.go:242
        	Error:      	Received unexpected error:
        	            	unexpected error while constructing env-aware sig configuration for AKSFlatcar: sig gallery configuration for AKSFlatcar not found
        	Test:       	Test_Windows23H2Gen2CachingRegression

```